config/app/prepends/ibexa/content_view.yaml

```diff
@@ -100,6 +100,26 @@ system:
                 match_all:
                     template: "@ibexadesign/content/views/link_embed.html.twig"
                     match: ~
+            download_inline:
+                ezbinaryfile_common:
```

Comment:
We can just name this `common`. No need for `ezbinaryfile_` prefix. Same goes for template file names.